package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node app.js",
     "lint": "eslint --ext .js,.json .",
-    "test": "NODE_ENV=test mocha --recursive test",
+    "test": "snyk test && NODE_ENV=test mocha --recursive test",
```

Comment:
Is there a specific reason for adding `snyk test` to the `test` script? I would prefer it to not be done there as it is a relatively expensive operation and one that we are moving towards doing on `prepush` only